src/components/Message/styles/Embed.css.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import baseStyles from '../../../styles/resets/baseStyles.css.js'
+import { BEM } from '../../../utilities/classNames'
+import { getColor } from '../../../styles/utilities/color.js'
+import { noteBoxShadow } from '../../../styles/mixins/noteStyles.css'
+
+const config = {
+  embedBorderRadius: '3px',
+  embedWidth: '9999px',
+  embedMaxWidth: '300px',
```

Comment:
I noticed a responsive issue for smaller viewports (mobile).
If you adjust to `100%`, then it (and the contents within) should be responsive:

![screen recording 2018-08-10 at 05 40 pm](https://user-images.githubusercontent.com/2322354/43982604-92db583e-9cc4-11e8-8ce3-03571013c4ee.gif)